pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -41,9 +43,16 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
+    for file in $(find -name \*.py); do
+      substituteInPlace "$file" \
+        --replace "/usr/bin/env python3" \
+                  "${python3}/bin/python3"
+    done
+    for file in $(find -name install-\*.sh); do
+      patchShebangs "$file"
+    done
```

Comment:
You can replace this by `patchShebangs .`, which  fixes all the shell scripts and python scripts at once. That is ok because all of these scripts are used only at build time.